### PR TITLE
Fix build after latest PyTorch changes

### PR DESCRIPTION
**Pull request by asuhan**

### CODE REVIEW

**Comment by asuhan** [comment]
Need to adjust for https://github.com/pytorch/pytorch/commit/82aa51114677b4d2187e1cc2d36f32158b7dddc0.

File: torch_xla/csrc/translator.cpp [hunk 1]
```diff
@@ -532,11 +530,35 @@ void TranslateSum(const torch::jit::Node* node, ComputationContext* cctx,
...
+      break;
+    }
+    case at::aten::nll_loss_forward: {
+      xla::Shape logits_shape = XlaHelpers::ShapeOfXlaOp(logits);
+      XLA_CHECK_EQ(logits_shape.rank(), 2);
+      // TODO(asuhan): Figure out what to do with total weight on the tracing
+      // JIT path.
+      xla::Shape total_weight_shape = xla::ShapeUtil::MakeShape(
+          xla::PrimitiveType::F32, {logits_shape.dimensions(0)});
+      xla::XlaOp total_weight =
```
Comment (dlibenzi) [inline torch_xla/csrc/translator.cpp]: IIRC, total weight had to be init to 1

File: torch_xla/csrc/translator.cpp [hunk 2]
```diff
@@ -532,11 +530,35 @@ void TranslateSum(const torch::jit::Node* node, ComputationContext* cctx,
...
+          XlaHelpers::ScalarBroadcast(0, total_weight_shape, b);
+      XLA_CHECK_EQ(node_outputs.size(), 2);
+      cctx->AddNodeOpById(node_outputs[1]->unique(), total_weight);
+      break;
+    }
+    default: {
```
Comment (dlibenzi) [inline torch_xla/csrc/translator.cpp]: I'd leave the brackets only where they are needed.

File: torch_xla/csrc/translator.cpp (hunk 1 above)
Comment (asuhan) [reply torch_xla/csrc/translator.cpp]: Done.

File: torch_xla/csrc/translator.cpp (hunk 2 above)
Comment (asuhan) [reply torch_xla/csrc/translator.cpp]: Done.

**Pull request merged**